### ISSUE TITLE
Advice on Using Moleculo Reads for Scaffolding with Spades

### ISSUE THREAD

**Issue by tarunaaggarwal** [ablab/spades#489]
Hi Spades makers and users,

I'm seeking some advice regarding whether or not to use Moleculo reads with Miseq and Hiseq short reads (both from Illumina) to improve my assembly. I have a good assembly already for my bacterium, but with lots of contigs (302). I have low coverage Moleculo reads for this bug that I want to use to either 1. assemble a new genome or 2. scaffold my contigs to reduce their number. 

However, I read that Spades is not recommended for low coverage PacBio reads so I'm thinking that also might apply to my Moleculo reads since they too are low coverage (less than 2x based on my calculations). 

Does anyone have suggestions on whether it is even worth my time to explore options 1 and/or 2?

Thank you so much!

**Comment by asl** [comment]
Well, if your reads are only 2x then you're out of luck with almost any method. You could try to scaffold though.

**Comment by tarunaaggarwal** [comment]
I totally agree. Some may even say I'm SOL. The low coverage was one of the original reasons why my PI never used that data, but I was hoping to make magic with it. Thank you so much for your input, Anton! I'll keep the issue open this week in case others have different thoughts.
